pkgs/development/python-modules/zeep/default.nix

```diff
@@ -29,6 +27,7 @@
 buildPythonPackage rec {
   pname = "zeep";
   version = "4.2.1";
+  format = "setuptools";
```

Comment:
From the source code, it looks like it uses pyproject.toml already to declare its build system.

Reply:
Unfortunately that did not work, because it thought I was doing a user installation for some reason.

Reply:
Such is life.